### PR TITLE
Disable hcalLaserEventFilter occupancy filter 

### PR DESCRIPTION
**Pull request by jkunkle**
We found that the occupancy filter in the hcalLaserEventFilter is erroneously removing good events.  We think that increasing the threshold to 8000 would be sufficient, but we also simply disable it in favor of the LaserMon filter.  We use the eras to ensure that the settings are only updated for 2018 data.

@deguio @mariadalfonso

### CODE REVIEW

**Comment by jkunkle** [comment]
To test one can run these steps

```
cmsrel CMSSW_10_2_X_2018-06-14-1100 # or a reasonably new release
cd CMSSW_10_2_X_2018-06-14-1100/src/
cmsenv
git cms-merge-topic 23597
cp -r /afs/cern.ch/user/j/jkunkle//public/HcalLaserCodeTest2/* .
scram b -j8
cmsDriver.py step1 --conditions 101X_dataRun2_Prompt_v9 -s RAW2DIGI,RECO --data --era Run2_2018 --eventcontent RECO,AOD --datatier RECO,AOD --filein root://eoscms.cern.ch//eos/cms/store/data/Run2018A/JetHT/RAW/v1/000/316/995/00000/FAEDC5AE-6F62-E811-AB45-02163E01A095.root  -n 100   --customise Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2018
cmsDriver.py step2 --filein file:step1_RAW2DIGI_RECO.root  --data --eventcontent MINIAOD --runUnscheduled --datatier MINIAOD --conditions 101X_dataRun2_Prompt_v9  --step PAT   --era Run2_2018 -n 100
cmsRun config_test.py
```
At the end you should see an output like

```
N events that Pass METFilter and Pass Charge cut = 100 -- GOOD!
N events that Fail METFilter and Fail Charge cut = 0 -- BAD!
N events that Pass METFilter and Fail Charge cut = 0 -- GOOD!
N events that Fail METFilter and Pass Charge cut = 0 -- GOOD!
```
This is the expected output since the laser was not firing.  If instead one comments out the changes here and reruns

```
cmsDriver.py step2 --filein file:step1_RAW2DIGI_RECO.root  --data --eventcontent MINIAOD --runUnscheduled --datatier MINIAOD --conditions 101X_dataRun2_Prompt_v9  --step PAT   --era Run2_2018 -n 100
cmsRun config_test.py
```
You see a different output with most events failing the MET filter

**Comment by perrotta** [comment]
please test

File: RecoMET/METFilters/python/hcalLaserEventFilter_cfi.py [hunk 1]
```diff
@@ -7,3 +7,10 @@
     badEvents=[]
 
 hcalLaserEventFilter = hcallaserevent.clone(BadRunEventNumbers=cms.vuint32(badEvents) )
+
+from Configuration.Eras.Modifier_run2_HCAL_2018_cff import run2_HCAL_2018
+run2_HCAL_2018.toModify( hcalLaserEventFilter,
+                           vetoByHBHEOccupancy = cms.bool( False),
```
Comment (perrotta) [inline RecoMET/METFilters/python/hcalLaserEventFilter_cfi.py]: Please drop type specification: types were already defined, and removing it here may help catching possible mistypes

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- One MET filter is disabled, as it was found to remove too many events (its threshold was already adjusted in previous updates of that filter, now they decide to disable it at all while waiting for further studies to decide an optimal one)
- No effect is visible in the limited stat of the jenkins tests (no laser firing in the tested samples)

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**